spec/controllers/concerns/json_api_sort_param_spec.rb

```diff
@@ -1,7 +1,20 @@
 require "rails_helper"
 
 RSpec.describe JsonApiSortParam do
-  let!(:controller) { Class.new { include JsonApiSortParam }.new }
+  let!(:controller) do
+    Class.new(Api::V0::ApiController) { include JsonApiSortParam }.new
```

Comment:
this change was required (instead of Class.new) since you're now relying on Controller behavior, like having a `params` method. Makes sense.

The existing tests (with an explicit param string) were left unchanged - also makes sense.

Reply:
Yes, I wanted to test both the default and explicit param cases and the explicit params are also easier to test.